<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signin</class>
 <widget class="QMainWindow" name="signin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>375</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(145, 206, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_p_signin" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>861</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLineEdit" name="lineEdit_signin_name">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_signin_password">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>100</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_signin_REpassword">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>190</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_safe_signin">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>280</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_Poetry">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>420</y>
      <width>881</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/image/qt_Poetry2.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_ok_signin">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(9, 0, 137);
image: url(:/new/prefix1/image/rang2.png);
font: 700 11pt &quot;Segoe UI&quot;;
background-color: rgb(0, 96, 202);

</string>
     </property>
     <property name="text">
      <string>تایید</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_menu_signin_p">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(9, 0, 137);
font: 700 11pt &quot;Segoe UI&quot;;
background-color: rgb(0, 96, 202);

</string>
    </property>
    <property name="text">
     <string>صفحه اول</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_English_signin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(9, 0, 137);
font: 700 11pt &quot;Segoe UI&quot;;
background-color: rgb(0, 96, 202);

</string>
    </property>
    <property name="text">
     <string>English</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Persian_signin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(9, 0, 137);
font: 700 11pt &quot;Segoe UI&quot;;
background-color: rgb(0, 96, 202);

</string>
    </property>
    <property name="text">
     <string>فارسی</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_menu_signin_e">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(9, 0, 137);
font: 700 11pt &quot;Segoe UI&quot;;
background-color: rgb(0, 96, 202);

</string>
    </property>
    <property name="text">
     <string>menu</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_safe_signin">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>661</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QFrame" name="frame_safe_2_signin">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(145, 206, 255);
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_safe_3_signin">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(145, 206, 255);
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_safe_4_signin">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(145, 206, 255);
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_show_safe_signin">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(9, 0, 137);
font: 700 11pt &quot;Segoe UI&quot;;
background-color: rgb(0, 96, 202);

</string>
     </property>
     <property name="text">
      <string>نمایش عدد</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_safe_1_signin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(145, 206, 255);
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_show_safe_signin_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>30</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(9, 0, 137);
font: 700 11pt &quot;Segoe UI&quot;;
background-color: rgb(0, 96, 202);

</string>
     </property>
     <property name="text">
      <string>show number</string>
     </property>
    </widget>
   </widget>
   <zorder>frame_Poetry</zorder>
   <zorder>widget_p_signin</zorder>
   <zorder>pushButton_menu_signin_p</zorder>
   <zorder>pushButton_English_signin</zorder>
   <zorder>pushButton_Persian_signin</zorder>
   <zorder>pushButton_menu_signin_e</zorder>
   <zorder>groupBox_safe_signin</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
